packages/dcos-integration-test/extra/test_dcos_diagnostics.py

```diff
@@ -651,9 +648,11 @@ def validate_unit(unit):
 
     # id, name, health, description, help should not be empty
     assert unit['id'], 'id field cannot be empty'
-    assert unit['name'], 'name field cannot be empty'
+    assert unit['name'], 'name field cannot be empty in {}'.format(unit)
     assert unit['health'] in [0, 1], 'health must be 0 or 1'
-    assert unit['description'], 'description field cannot be empty'
+
+    # Green painting. The Docker service has no description.
+    assert unit['description'] or unit['name'] == 'docker', 'description field cannot be empty'
```

Comment:
So, this is up for @janisz and @DominikDary to decide. The Docker Windows service has no description. IMHO it should not be part of the service list anyways as it is not in DC/OS. I am not sure how we setup the service. Maybe there is a way to set the description.

Reply:
I'm fine removing docker from the `servicelist.txt` file.

Reply:
Done.

Reply:
I'm fine with this approach. descriptions are only for users so it's fine that docker has no description

Reply:
I dropped `docker` from the services list.